src/physics/em/detail/MollerEnergyDistribution.i.hh

```diff
@@ -25,7 +25,7 @@ MollerEnergyDistribution::MollerEnergyDistribution(real_type electron_mass_c_sq,
                                                    real_type inc_energy)
     : inc_energy_(inc_energy)
     , total_energy_(inc_energy + electron_mass_c_sq)
-    , min_energy_fraction_(min_valid_energy / inc_energy)
+    , min_energy_fraction_(0.5 * min_valid_energy / inc_energy)
```

Comment:
After all the discussion (making sure your initial suggestion was right and going back to the code and the physics) I actually think we *should* keep your original suggestion and pass around the *production cutoff*. That's the value that's used in the sampler and is easiest to follow. It's confusing to double the value in the Interactor constructor, then halve it here.

Reply:
Yeah, that seems clearer, reverted. I don't think my original fix was quite right (not accounting for the factor of two already being in the production cuts but not in `min_valid_energy`, as you pointed out), but it should be correct now.